bin/cycle_logs.rb

```diff
@@ -17,15 +17,23 @@
     end
   else
 		all_logs.select {|l| l =~ /#{i}/}.each do |log|
-      basename = log.split('.').first
+      if log.split('.').first == 'delayed_job' && log.split('.').size == 3
```

Comment:
this could use a comment explaining the intent

Reply:
Sure - it has to do with Scott renaming the `delayed_job` logfiles to have the environment name in them (made testing easier locally).